includes/class-sensei-frontend.php

```diff
@@ -1320,17 +1320,8 @@ public function sensei_lesson_meta( $post_id = 0 ) {
 	} // sensei_lesson_meta()
 
 	public function sensei_lesson_preview_title_text( $course_id ) {
-
 		$preview_text = __( 'Preview', 'woothemes-sensei' );
 
-		// if this is a paid course.
-		if ( Sensei_WC::is_woocommerce_active() ) {
-			$wc_post_id = get_post_meta( $course_id, '_course_woocommerce_product', true );
-			if ( 0 < $wc_post_id ) {
-				$preview_text = __( 'Free Preview', 'woothemes-sensei' );
-			} // End If Statement
-		}
-
 		/**
 		 * The lesson preview indicator text. Defaults to "Preview" or "Free
```

Comment:
Should we remove the references to "Free Preview" from this comment?

Reply:
Good catch. I made the change and committed to `release/2.0`